ingestion/examples/workflows/bigquery.json

```diff
@@ -6,8 +6,17 @@
       "host_port": "bigquery.googleapis.com",
       "username": "gcpuser@project_id.iam.gserviceaccount.com",
       "service_name": "gcp_bigquery",
-      "options": {
-        "credentials_path": "examples/creds/bigquery-cred.json"
+      "credentials":{
```

Comment:
instead of creating another dictionary here, prefer it having it under "options" key. That way we are not deviating from the SQLConnectionConfig